code/game/objects/explosion.dm

```diff
@@ -64,14 +64,14 @@
 					else if(dist <= far_dist)
 						var/far_volume = Clamp(far_dist, 30, 50) // Volume is based on explosion size and dist
 						far_volume += (dist <= far_dist * 0.5 ? 50 : 0) // add 50 volume if the mob is pretty close to the explosion
-						if(devastation_range > 0)
+						if(devastation_range > 2)
 							M.playsound_local(epicenter, 'sound/effects/explosionfar.ogg', far_volume, 1, frequency, falloff = 5)
 							shake_camera(M, 3, 1)
 						else
 							M.playsound_local(epicenter, 'sound/effects/explosionsmallfar.ogg', far_volume, 1, frequency, falloff = 5)
 							skip_shake = 1
 
-					if(!explosion_shake_message_cooldown && devastation_range > 0 && !skip_shake)
+					if(!explosion_shake_message_cooldown && devastation_range > 2 && !skip_shake)
 						M << "<span class='danger'>You feel the station's structure shaking all around you.</span>"
```

Comment:
This needs a  check for if you're actually in the same station/ship, conscious, etc. It also needs to be more generic (or more specific, like for planet tiles).